File: assignment18.py
list_of_nums = [19, 19, 5,5,5]

def numTracker(list):
    num_of_19 = 0
    num_of_5 = 0
    for item in list:
        if item == 19:
            num_of_19 += 1
        if item == 5:
            num_of_5 += 1
    if num_of_19 == 2 and num_of_5 <= 3:
        print(True)
    else:
        print(False)

numTracker(list_of_nums)

File: javascript_assignments/assignment18.js
list_of_nums = [19,19,5,5,5]

function numTracker(list){
    let num_of_19s = 0
    let num_of_5s = 0

    for (let i = 0; i < list.length; i++){
        if (list[i] == 19){
            num_of_19s += 1
        }
        if (list[i] == 5){
            num_of_5s += 1
        }
    }
    if (num_of_19s == 2 && num_of_5s <= 3){
        console.log(true)
    } else{
        console.log(false)
    }
}

numTracker(list_of_nums)